Makefile

```diff
@@ -392,7 +392,7 @@ currentversion:
 
 test: $(LINUXKIT) test-images-patches | $(DIST)
 	@echo Running tests on $(GOMODULE)
-	$(QUIET)$(DOCKER_GO) "gotestsum --jsonfile $(DOCKER_DIST)/results.json --junitfile $(DOCKER_DIST)/results.xml --raw-command -- go test -coverprofile=coverage.txt -covermode=atomic -race -json ./..." $(GOTREE) $(GOMODULE)
+	$(QUIET)$(DOCKER_GO) "gotestsum --jsonfile $(DOCKER_DIST)/results.json --junitfile $(DOCKER_DIST)/results.xml --raw-command -- go test -tags kubevirt -coverprofile=coverage.txt -covermode=atomic -race -json ./..." $(GOTREE) $(GOMODULE)
```

Comment:
@christoph-zededa  can you review this test thing? I don't know what the tags do.

Reply:
Then I would review my own code: https://github.com/lf-edge/eve/pull/3768#issuecomment-1966813344 ;-)

It is passing the build flag for kubevirt to build the tests with this flag set.

Reply:
:-) :-)
So the assumption is that with kubevirt virtualization mode we'll run a superset of the tests, which is presumably true now. 
Down the road we might have different subsets of tests for different variants, but we can cross that bridge when we get to it.